src/app/app-modules/nurse-doctor/nurse-worklist-tabs/nurse-worklist-tabs.component.html

```
@@ -1,11 +1,11 @@
-<div class="container-fluid">
-  <mat-tab-group class="tabs_css">
-    <mat-tab [label]="currentLanguageSet?.nurse_tabs?.visit">
+<div class="px-6 pt-3">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)